guide/getting-started/hardware.md

```diff
@@ -145,6 +147,26 @@ Bitcoin ATMs are usually bound to local money transmission laws and regulations,
 
 You can find local ATMs using services like [coinatmradar](https://coinatmradar.com/).
 
+## Point of sale
+
+People looking to accept bitcoin payments, especially when using the lightning network, sometimes use dedicated point of sale (POS) terminals when selling goods and services.
+
+{% include picture.html
+   image = "/assets/images/guide/getting-started/hardware/point-of-sale.jpg"
+   retina = "/assets/images/guide/getting-started/hardware/point-of-sale@2x.jpg"
+   mobile = "/assets/images/guide/getting-started/hardware/point-of-sale-mobile.jpg"
+   mobileRetina = "/assets/images/guide/getting-started/hardware/point-of-sale-mobile@2x.jpg"
+   alt-text = "Examples of point of sale hardware devices."
+   width = 800
+   height = 400
+%}
+
+Dedicated bitcoin POS hardware is used to request and receive bitcoin payments in-person. These act very similar to traditional POS systems but present to payees a bitcoin on-chain, lightning, or [unified](https://bitcoinqr.dev/) payment request. Payments can also be received over NFC.
+
+As a user needs to be online to receive a lightning payment, POS terminals can act as a always on lightning node allowing lightning payments to be received easily.
+
+Accepting lightning payments with POS hardware can also be offline using a standard like [LNURL](https://github.com/fiatjaf/lnurl-rfc) to easily receive payments. An example of this is [LNURLPoS](https://github.com/arcbtc/LNURLPoS)
```

Comment:
I agree with everything being said here, but I think this and the above paragraph make it seem like there are two ways to make a bitcoin POS:
- Always on lightning node embedded in POS terminal
- Completely offline LNURLPoS

In my experience, these are the least common bitcoin POS systems. The most common bitcoin POS system is:
- POS terminal which connects remotely to a lightning node with API calls (e.g. using Chivo, Ibex, or OpenNode, or even accessing one's BTCPayServer from an iPad).

Reply:
Made changes to reflect these points, though your last point would probably be more suitable (and already is very minimally) in the software categories page.